doc/ref_model/chapters/chapter03.md

```diff
@@ -539,6 +542,150 @@ SyncE was standardized by the ITU-T, in cooperation with IEEE, as three recommen
+----------------|------------
+[Ingress:](https://kubernetes.io/docs/concepts/services-networking/ingress/) | Ingress is a collection of rules that allow inbound connections to reach the endpoints defined by a backend. An Ingress can be configured to give services externally-reachable urls, load balance traffic, terminate SSL, offer name based virtual hosting etc.
+[Service:](https://kubernetes.io/docs/concepts/services-networking/service/) | Service is a named abstraction of software service (for example, mysql) consisting of local port (for example 3306) that the proxy listens on, and the selector that determines which pods will answer requests sent through the proxy.
+[EndpointSlices:](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/) | Endpoints and Endpointslices are a collections of endpoints that contains the ip address, v4 and v6, of the pods that represents a service.
+[Network Policy:](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/) | NetworkPolicy defines which network traffic is allowed to ingress and egress from a set of pods.
+
+There is no need to explicitly define internal load balancers, servers pools, service monitors, firewalls and so on. The kubernetes sematics and relation between the different objects defined in the object manifests contains all the information needed.
+
+Example: The manifests for service *my-service* and the*deployment* with the four load balanced pods of type *my-app*
+
+Service:
+```
+apiVersion: v1 
+kind: Service 
+metadata: 
+        name: my-service
+        spec: 
+                selector:		
+                        app: my-app	
+                ports: 
+                        - protocol: TCP
+                                port: 123
+```
+Deployment:
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata: name: my-app-deployment
+spec:
+        selector:
+                matchLabels:
+                        app: my-app
+                        replicas: 4
+                        template:
+                                metadata:
+                                        labels:
+                                                app: my-app
+                        spec:
+                                containers:
+                                        - name: my-app
+                                          image: my-app-1.2.3
+                                          ports:        
+                                          - containerPort: 123 
+```
+
+This is all that is needed to deploy 4 pods/containers that are fronted by a service that performe sload balancing. The *Deplyment* will ensure that there are always four pods of type *my-app* available. the *deplyment* is responsible for the full lifecycle management of the pods, this includes in service update/upgrade.
+
+None of this is of much help however when implementing network service functions such as VNFs/CNFs that requires multi networking and network orchestartion
```

Comment:
"that requires multi networking and network orchestartion"  ---> "that require multi networking and network orchestration."